src/api/openapi.yaml

```diff
@@ -1513,6 +1514,7 @@ components:
                     $ref: '#/components/schemas/ResolutionType'
                   systems:
```

Comment:
Should add a `maxItems` here too.   I get that we're limiting the number of 'distinct' systems but there can still never be more than 50 systems in any one issue...